core/controlplane/config/templates/cloud-config-worker

```diff
@@ -1099,14 +1099,27 @@ write_files:
       # Create the device node for the nvidia-uvm module
       ACTION=="add" DEVPATH=="/module/nvidia_uvm" SUBSYSTEM=="module" RUN+="/opt/nvidia/current/bin/create-uvm-dev-node.sh"
 
+  - path: /opt/nvidia-build/git_versions.sh
+    owner: root:root
+    permissions: 0755
+    content: |
+      #!/bin/bash
+      set -e
+      source /usr/share/coreos/release
+      OVERLAY_VERSION=`curl -s https://raw.githubusercontent.com/coreos/manifest/v${COREOS_RELEASE_VERSION}/release.xml | grep coreos-overlay | cut -d\" -f 8`
+      PORTAGE_VERSION=`curl -s https://raw.githubusercontent.com/coreos/manifest/v${COREOS_RELEASE_VERSION}/release.xml | grep portage-stable | cut -d\" -f 8`
+      git -C /var/lib/portage/coreos-overlay checkout ${OVERLAY_VERSION}
+      git -C /var/lib/portage/portage-stable checkout ${PORTAGE_VERSION}
+
   - path: /opt/nvidia-build/_container_build.sh
     owner: root:root
     permissions: 0755
     content: |
       #!/bin/sh
 
       # Default: use binary packages instead of building everything from source
-      EMERGE_SOURCE_FLAGS=gK
+      # Default: build everything from source.
+      #EMERGE_SOURCE_FLAGS=gK
```

Comment:
What pros and cons would there be by building everything from source instead of using binary packages?
Probably it now takes a lot more time for a GPU-enabled worker node to be up and running, at least?

Reply:
I was getting 403s when trying to grab the binary packages from the repos. At least with building from source it's consistent for me, in my testing.

Reply:
Ah, thanks for investigating it. Then, this change LGTM 👍 

Btw, would you mind sharing me the full log messages of the installation process which helps me which files are getting 403s, perhaps in a gist?
So that I can probably try to contact someone maintains the files to see what's going on.

Reply:
@mumoshu Sure, here you are: https://gist.github.com/kylegato/689a366b62f744702b634e710cd4cc76

Reply:
@kylegato I've reported the 403s issue to https://github.com/coreos/bugs/issues/2207 and it seems to have fixed now. In nutshell, it wasn't a planned or expected change.

Sorry for the back and forth but would you be willing to revert only the build fix part of this PR so that we can still benefit from prebuilt binaries for faster node startup?

Also, thank you very much for locating the issue 👍